portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,96 @@
+package com.google.sps.data;
+
+import javax.servlet.http.HttpServletRequest;
+import com.google.appengine.api.datastore.Entity;
+
+/**
+ * A class for storing a Comment.
+ */
+public class Comment {
+    private String author;
+    private String subject;
+    private String text;
+    private long timestamp;
+
+    /**
+     * No argument constructor.
+     * Set fields with setter methods.
+     */
+    public Comment() {
+        this.timestamp = System.currentTimeMillis();
```

Comment:
When would we want a comment without the author/subject/text fields already known by the caller?

Reply:
Good point, I don't think we would ever really want it. So I should use a constructor that initializes all required instance variables up front?